domains/nazmul.json

```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "seraprogrammer",
+        "email": "programmernazmul1@gmail.com",
+        "discord": "seraprogrammer"
+    },
+    "record": {
+        "TXT": "vc-domain-verify=nazmul.is-a.dev,741b4962f2334f48af82"
```

Comment:
This should be your actual A records, no vercel verification here. See docs: https://docs.is-a.dev/guides/vercel